src/GraphQL.Tests/StarWars/StarWarsIntrospectionTests.cs

```diff
@@ -746,7 +746,7 @@ public void full_schema_query()
                   'fields': [
                     {
                       'name': 'id',
-                      'description': null,
+                      'description': 'The id of the character.',
```

Comment:
Added missing descriptions for introspection tests. The unit tests seemed outdated.